test/utils/logger_utils.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+
+# Copyright 2024 IBM All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from http.client import HTTPConnection
+from ibm_cloud_sdk_core.logger import get_logger
+
+
+def setup_test_logger(level: int) -> logging.Logger:
+    """Sets up logging with a console handler to assist test cases."""
+    logger = get_logger()
+    logger.setLevel(level)
+    # if not logger.hasHandlers():
```

Comment:
Is this comment outdated?

Reply:
probably :)   I'll fix it before merging.